project_liberation/system_check.py

```diff
@@ -26,3 +33,23 @@ def check_google_api_key(app_configs: Any, **kwargs: Any) -> list:  # pylint: di
         return [GOOGLE_API_KEYS_ERRORS["length"]]
 
     return []
+
+
+def check_custom_storage_directories(setting_name: str) -> list:
+    if not hasattr(settings, setting_name):
+        return [setting_not_declared_error(setting_name)]
+
+    if not isinstance(settings.__getattr__(setting_name), str):
+        return [setting_not_a_string_error(setting_name)]
+
+    return []
+
+
+@register()
+def check_company_employees_storage(app_configs: Any, **kwargs: Any) -> list:  # pylint: disable=unused-argument
+    return check_custom_storage_directories("COMPANY_EMPLOYEES_STORAGE")
+
+
+@register()
+def check_testimonial_photos_storage(app_configs: Any, **kwargs: Any) -> list:  # pylint: disable=unused-argument
+    return check_custom_storage_directories("TESTIMONIAL_PHOTOS_STORAGE")
```

Comment:
You can create from these two system checks one with map to settings names which has to be checked.

Reply:
Done